spec/features/amusement_parks/show_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+
+RSpec.describe AmusementPark, type: :feature do
+  before(:each) do
+    @six_flags = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+  
+    @mechanic_1 = Mechanic.create!(name: "Tom Jones", years_experience: 18)
+    @mechanic_2 = Mechanic.create!(name: "Sandra Boop", years_experience: 10)
+  
+    @hurler = @six_flags.rides.create!(name: 'The Hurler', thrill_rating: 7, open: true)
+    @scrambler = @six_flags.rides.create!(name: 'The Scrambler', thrill_rating: 4, open: true)
+    @ferris = @six_flags.rides.create!(name: 'Ferris Wheel', thrill_rating: 7, open: false)
+  
+    @mr_1 = MechanicRide.create!(ride_id: @hurler.id, mechanic_id: @mechanic_1.id)
+    @mr_2 = MechanicRide.create!(ride_id: @scrambler.id, mechanic_id: @mechanic_1.id)
+    @mr_3 = MechanicRide.create!(ride_id: @ferris.id, mechanic_id: @mechanic_1.id)
+  
+    @mr_4 = MechanicRide.create!(ride_id: @scrambler.id, mechanic_id: @mechanic_2.id)
+    end
+  
+  it " has a show page with name, price for the park and all mechanics" do
+    visit "/amusement_parks/#{@six_flags.id}"
+
+    expect(page).to have_content("Six Flags")
+    expect(page).to have_content(75)
```

Comment:
THis assertion would be a bit more meaningful with a text label to anchor it! i.e. `expect(page). to have_content("Cost to enter: 75")`